src/Core/Domain/Feature/Command/BulkDeleteFeatureValueCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Feature\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\Feature\ValueObject\FeatureValueId;
+
+class BulkDeleteFeatureValueCommand
+{
+    /**
+     * @var FeatureValueId[]
+     */
+    public readonly array $featureValueIds;
+
+    /**
+     * @param int[] $featureValueIds
+     */
+    public function __construct(array $featureValueIds)
+    {
+        $ids = [];
+        foreach ($featureValueIds as $featureValueId) {
+            $ids[] = new FeatureValueId($featureValueId);
+        }
+
+        $this->featureValueIds = $ids;
```

Comment:
You could use an array_map to avoid the temp variable here but it's optional:
```suggestion
        $this->featureValueIds = array_map(static function(int $featureValueId) : FeatureValueId {
            return new FeatureValueId($featureValueId);
        }, $featureValueIds);
```